modules/service/query-service.js

```diff
@@ -6,21 +6,36 @@ class QueryService {
         this.logger = ctx.logger;
         this.networkService = ctx.networkService;
         this.validationService = ctx.validationService;
+        this.blockchainService = ctx.blockchainService;
         this.dataService = ctx.dataService;
         this.fileService = ctx.fileService;
         this.workerPool = ctx.workerPool;
     }
 
     async resolve(id, load, isAssetRequested, node) {
-        let result = await this.networkService.sendMessage('/resolve', id, node);
-        if (!result || (Array.isArray(result) && result[0] === "ack")) {
+        const resolvePromise = new Promise(async (resolve, reject) => {
+            const timer = setTimeout(() => {
+                resolve(null);
+            }, 3000);
```

Comment:
Maybe we can parametrize this and move to config? What do you think? Maybe someone would like to wait more

Reply:
Or to move to constants